.github/CONTRIBUTING.md

```diff
@@ -1,12 +1,29 @@
 # How to use Jedis Github Issue
 
-* Github issues SHOULD ONLY BE USED to report bugs, and for DETAILED feature requests. Everything else belongs to the Jedis Google Group.
+* Github issues SHOULD BE USED to report bugs and for DETAILED feature requests. Everything else belongs in the [Jedis Google Group](https://groups.google.com/g/jedis_redis) or [Jedis GitHub Discussions](https://github.com/redis/jedis/discussions).
 
-Jedis Google Group address:
-  
-https://groups.google.com/forum/?fromgroups#!forum/jedis_redis
+Please post general questions to Google Group or Github discussions. These can be closed without response when posted to Github issues.
 
-Please post General questions to Google Group. It can be closed without answer when posted to Github issue.
+# How to contribute by Pull Request
+
+1. Fork Jedis on github (https://help.github.com/articles/fork-a-repo/)
```

Comment:
```
[Fork](https://help.github.com/articles/fork-a-repo/) Jedis on github
```

Reply:
Be consistent with the beginning of the article, use `GitHub` instead of `github`

Reply:
@yangbodong22011 Changed the first (and only) `GitHub`